### ISSUE TITLE
Error booting 0.1.5-alpha release

### ISSUE THREAD

**Issue by joel-lilja** [eliasdoehne/stellaris-dashboard#11]
Hello I am getting an error booting after installing alpha release 0.1.5-alpha  

I subscribed to steam workshop and installed python 3.7  
The text of the error message is as follow   

>Configuration:
  save_file_path: C:\Users\joel\Documents\Paradox Interactive\Stellaris\save games
  base_output_path: C:\Users\joel\Documents\stellaristimeline
  threads: 5
  show_everything: False
  only_show_default_empires: True
  extract_system_ownership: True
2018-08-14 21:02:05,531 - stellarisdashboard.save_parser - INFO - Cython extensions not available, using slow parser. Error message: "cannot import name 'token_value_stream' from 'stellarisdashboard.cython_ext'

When i install from master a slightly more recent update this does not occur.

**Comment by eliasdoehne** [comment]
Yes, this error should be fixed in the proper release of v0.1.5. Cython is a really cool tool that allows to use precompiled binaries, which are much faster than regular python code. I include these binaries directly in the release because most people will have trouble installing a compiler on Windows. I think there was some issue with one of the binaries, so I just compiled those again last night. 
(When I was preparing the prerelease, I only had my laptop and couldn't fix this)

Just to confirm: If you use the most recent version in the master branch, there is no issue, right? And you are using Python 3.7 on Windows? That would be great because some other user reported issues with Python 3.7 so this is why I recommend using the older 3.6 instead

**Comment by moreorlessgit** [comment]
I've had this issue too, in the new build. Python 3.7 on Windows 10.

> Configuration:
  save_file_path: WindowsPath('C:/Users/Bryce/Documents/Paradox Interactive/Stellaris/save games')
  base_output_path: WindowsPath('C:/Stellaristimelineexport')
  threads: 3
  show_everything: False
  only_show_default_empires: False
  extract_system_ownership: True
2018-08-17 13:04:22,666 - stellarisdashboard.save_parser - INFO - Cython extensions not available, using slow parser. Error message: "cannot import name 'token_value_stream' from 'stellarisdashboard.cython_ext' (c:\stellaris-dashboard-cur\src\stellarisdashboard\cython_ext\__init__.py)"
2018-08-17 13:04:40,402 - stellarisdashboard.cli - INFO - Waiting for new saves in C:\Users\Bryce\Documents\Paradox Interactive\Stellaris\save games
  Serving Flask app "stellarisdashboard.dash_server" (lazy loading)
  Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
  Debug mode: off
2018-08-17 13:04:40,672 - werkzeug - INFO -  * Running on http://127.0.0.1:28053/ (Press CTRL+C to quit)
2018-08-17 13:16:01,299 - stellarisdashboard.save_parser - INFO - Found 1 new files: autosave_2210.01.01...
2018-08-17 13:16:01,299 - root - INFO - Parsing Save File C:\Users\Bryce\Documents\Paradox Interactive\Stellaris\save 
games\borgempire_2025805131\autosave_2210.01.01.sav...

It recognizes my save game, but it doesn't finish parsing it and doesn't create the DB file.

I've attached the save game itself.

[borgempire_2025805131.zip](https://github.com/eliasdoehne/stellaris-dashboard/files/2298158/borgempire_2025805131.zip)
